pdns/packethandler.hh

```diff
@@ -108,6 +108,7 @@ private:
   bool d_logDNSDetails;
   bool d_doIPv6AdditionalProcessing;
   bool d_doDNAME;
+  bool d_doExpandALIAS;
```

Comment:
I wonder if these flags, in general, shouldn't be static so we just parse them once on startup. Not a job for this PR though.